src/TwigComponent/tests/Unit/Attribute/AsTwigComponentTest.php

```diff
@@ -21,6 +21,41 @@
  */
 final class AsTwigComponentTest extends TestCase
 {
+    /**
+     * @dataProvider provideServiceConfigData
+     */
+    public function testServiceConfigValues(AsTwigComponent $attribute, array $expectedConfig): void
+    {
+        $this->assertSame($expectedConfig, $attribute->serviceConfig());
+    }
+
+    public static function provideServiceConfigData(): iterable
+    {
+        yield 'No values' => [
+            new AsTwigComponent(),
+            [
+                'expose_public_props' => true,
+                'attributes_var' => 'attributes',
+            ],
+        ];
+        yield 'Default values' => [
+             new AsTwigComponent(null, null, true, 'attributes'),
+             [
+                 'expose_public_props' => true,
+                 'attributes_var' => 'attributes',
+             ],
+        ];
+        yield 'Name and template set' => [
+             new AsTwigComponent('foo', 'template'),
+             [
+                'key' => 'foo',
+                'template' => 'template',
```

Comment:
```suggestion
                 'key' => 'foo',
                 'template' => 'template',
```

Reply:
Thanks.. i'll see why the CI did not get those..

Reply:
Well `array_indententation` is not in the Symfony rules https://cs.symfony.com/doc/rules/whitespace/array_indentation.html